RadixWallet/Features/AccountAndPersonaHidingFeature/AccountAndPersonaHiding+View.swift

```diff
@@ -0,0 +1,93 @@
+extension AccountAndPersonaHiding.State {
+	var viewState: AccountAndPersonaHiding.ViewState {
+		.init(
+			hiddenAccountsCount: hiddenEntitiesStats?.hiddenAccountsCount ?? 0,
+			hiddenPersonasCount: hiddenEntitiesStats?.hiddenPersonasCount ?? 0
+		)
+	}
+}
+
+extension AccountAndPersonaHiding {
+	public struct ViewState: Equatable {
+		public let hiddenAccountsCount: Int
+		public let hiddenPersonasCount: Int
+
+		public var hiddenAccountsText: String {
+			if hiddenAccountsCount == 1 {
+				L10n.AppSettings.EntityHiding.hiddenAccount(1)
```

Comment:
Why does this string take an argument?

Reply:
rather this while function could be: 

```swift
public var hiddenAccountsText: String {
    L10n.AppSettings.EntityHiding.hiddenAccount(hiddenAccountsCount)
}
```

right?

Reply:
This is how currently the string is added in Crowding, was done by Android, will update once the string is also updated.

@CyonAlexRDX about just having:
```
public var hiddenAccountsText: String {
    L10n.AppSettings.EntityHiding.hiddenAccount(hiddenAccountsCount)
}
```
I am not sure if Crowding + L10n can handle the multiplicity with just one function, but probably it can, don't know.